boskos/mason/mason.go

```diff
@@ -34,6 +34,7 @@ const (
 // Masonable should be implemented by all configurations
 type Masonable interface {
 	Construct(context.Context, common.Resource, common.TypeToResources) (*common.UserData, error)
+	Destruct(context.Context, *common.Resource, interface{}) error
```

Comment:
following on that comment - does `Destruct` even need to be passed the leased resources? Can't it just get them from the resource itself?

Reply:
Good point, removed.